Server/mods/deathmatch/local.conf

```diff
@@ -274,6 +274,12 @@
          Values: 0 - Off, 1 - Enabled.  Default - 0 -->
     <elementdata_whitelisted>0</elementdata_whitelisted>
 
+    <!-- This parameter determines that the server checks when the players connect that there are no other players with the same serial number. 

```

Comment:
I think you also need to add this to editor.conf

It's really annoying how everything is repeated in mtaserver.conf, mtaserver.conf.template, local.conf and editor.conf. We should refactor this in a future PR.

Reply:
Thx